### DISCUSSION TITLE
Access Feature Toggles

### DISCUSSION THREAD

**Discussion by mjversagility** [solidify/scale#9] [General]
How do you access the feature toggles? I am interesting in looking at the updated program board with the capacity and load indicators.

**Comment by MOlausson** [comment]
We just released https://github.com/solidify/scale/releases/tag/v1.2.123 where the new board can be tested by adding #new-board to the end of the Azure DevOps url. We'll keep it behind the flag for a couple of weeks, feedback is appreciated.

**Comment by mjversagility** [comment]
Thank you so much Mathias,

We did notice one thing. On the new board with respect to Capacity. The capacity we define is not persisting. The next time we come back to the board its gone.

Regards,
Mike

**Reply by MOlausson** [reply]
Thanks for the feedback Mike, we'll look into it.
